pkg/cmd/certinspection/certinspection.go

```diff
@@ -106,8 +108,8 @@ func (o *CertInspectionOptions) Run() error {
 			inspectConfigMap(castObj)
 		case *corev1.Secret:
 			inspectSecret(castObj)
-		case *certificatesv1beta1.CertificateSigningRequest:
-			inspectCSR(castObj)
+		// case *certificatesv1beta1.CertificateSigningRequest:
```

Comment:
huh, I forgot to move this?

Reply:
re-adding this in https://github.com/openshift/library-go/pull/1581 (also with v1beta1 and v1 support)